src/api/app/views/notifications/_group_action.text.erb

```diff
@@ -0,0 +1 @@
+- <%= action.inspect %>
```

Comment:
Is this `.inspect` intended?

Reply:
Yes. These notification templates were copied from `app/views/event_mailer/` to `app/views/notifications/`, and adapted to use the new polymorphic notification columns. In [this case](https://github.com/openSUSE/open-build-service/blob/d61545cf84f10b93b299a5d4e9e8992164789963/src/api/app/views/event_mailer/_group_action.text.erb#L1) it was not needed to adapt anything, and only a copy was made.

Reply:
okay